### DISCUSSION TITLE
Welcome to cij Discussions!

### DISCUSSION THREAD

**Discussion by singularitti** [MineralsCloud/cij#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rainbowzzzzz** [comment]
Hi, how can I get the input01 file based on the phonon frequency file obtained from the qha calculation?

**Reply by chazeon** [reply]
Input01 is almost the same file as what you use for the QHA calculation

**Reply by Rainbowzzzzz** [reply]
Thank you very much for your response. I'm unsure how to sort the phonon mode frequencies as described in Appendix B, and also how to obtain the weights for different q points？

**Reply by chazeon** [reply]
You will need to get the eigenvectors first. I don't know which software you are using, phonopy and QE has their own option to enable eigenvector output. Then you need to write your own wrapper and use the sorting from https://github.com/MineralsCloud/cij/blob/dev/cij/misc/evec_sort.py
For many materials sorting or not will not change the final result much.
